docs/baking.md

```diff
@@ -105,50 +114,27 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node-<network> snapshot import <path to the snapshot file>
 ```
 
-### Starting the node
+## Setting up baker and endorser key
 
-After the snapshot import, you can finally start the node by running:
+Note that account activation from JSON file and baker registering require
+running fully-bootstrapped `tezos-node`. In order to start node service do the following:
 ```
 sudo systemctl start tezos-node-<network>.service
 ```
 
-Note that even after the snapshot import the node can still be out of sync. It may require
+Even after the snapshot import the node can still be out of sync. It may require
 some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
-you can use `tezos-client`:
+you can use tezos-client:
 ```
 sudo -u tezos tezos-client bootstrapped
 ```
 
-To stop the node, run:
-```
-sudo systemctl stop tezos-node-<network>.service
-```
-
-If you want the service to start automatically at boot, use `enable`:
-```
-sudo systemctl enable tezos-node-edo2net.service
-```
-and to disable it from doing so, instead use:
-```
-sudo systemctl disable tezos-node-edo2net.service
-```
-
-You can check node logs via `journalctl`:
-```
-journalctl -u tezos-node-<network>.service
-```
-
-## Setting up baker and endorser daemons
-
-### Setting up daemon data directories
-
-Data directories for baker and endorser daemons are defined in the
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`.
-By default, both these daemons have `/var/lib/tezos/.tezos-client` set as a `DATA_DIR`.
-
-Additionally, you need to specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
-to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
+By default `tezos-baking-<network>.service` will use the `baker` alias for the
+key that will be used for baking and endorsing.
 
+In case you want to use different alias for the baking account, replace `baker` with
```

Comment:
I would move instructions for customizations like  to the "Advanced" section at the very end of the document. There is no reason for users to think about this, know about this or change this, ever. Of course there will be users who want to do this anyway, so... Advanced section :)